system_metrics_collector/launch/system_cpu_and_memory.launch.py

```diff
@@ -12,24 +12,75 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Launch system CPU and system memory metrics collector nodes."""
-
-
 from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
 from launch_ros.actions import Node
 
+# Launch argument names
+CPU_NODE_NAME = 'linux_cpu_collector_node_name'
+MEMORY_NODE_NAME = 'linux_memory_collector_node_name'
+MEASUREMENT_PERIOD = 'measurement_period'
+PUBLISH_PERIOD = 'publish_period'
+PUBLISH_TOPIC = 'publish_topic'
+
+# Default argument values
+default_cpu_node_name = 'linux_cpu_collector'
+default_memory_node_name = 'linux_memory_collector'
+default_measurement_period = '1000'
+default_publish_period = '60000'
```

Comment:
variables containing a scientific value should include the unit `default_measurement_period` -> `default_measurement_period_in_ms`